metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -311,6 +311,20 @@ class Compilers(
       pc.signatureHelp(CompilerOffsetParams.fromPos(pos, token)).asScala
     }.getOrElse(Future.successful(new SignatureHelp()))
 
+  def classForPosition(
```

Comment:
Not sure about the time, maybe `enclosingClass` would be better.

Reply:
`enclosingClass` definitely sounds better

Reply:
changed!